timepiece/entries/views.py

```diff
@@ -1004,3 +1004,13 @@ def activity_cheat_sheet(request):
     return render(request, 'timepiece/entry/activity_cheat_sheet.html', {
         'activities': Activity.objects.all().order_by('name'),
     })
+
+@login_required
+def get_active_entry(request):
+    active_entry = utils.get_active_entry(request.user)
+    if active_entry:
+        active_entry_json = {
+            'start_time': active_entry.start_time,
+            'project': active_entry.project.name
+        }
+        return JsonResponse(active_entry_json, safe=False)
```

Comment:
Did you need the `safe=False` here?  Since it is a `dict`, I would assume you would not need it.